t/ui/13-admin.t

```diff
@@ -581,7 +581,7 @@ subtest 'edit job templates' => sub() {
     wait_for_ajax;
     like($result->get_text(), qr/YAML saved!/, 'saving confirmed') or diag explain $result->get_text();
     $yaml = $driver->execute_script('return editor.doc.getValue();');
-    is($yaml, "products: {}\nscenarios: {}", 'YAML was reset to default') or diag explain $yaml;
+    is($yaml, "products: {}\nscenarios: {}\n", 'YAML was reset to default') or diag explain $yaml;
```

Comment:
I'd like a test that asserts the added line-break on a non-default template.

Reply:
done